manifests/z/zurawiki/gptcommit/0.5.7/zurawiki.gptcommit.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: zurawiki.gptcommit
+PackageVersion: 0.5.7
+InstallerType: zip
+Installers:
+- Architecture: x64
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: .\gptcommit.exe
+    PortableCommandAlias: gptcommit
+  InstallerUrl: https://github.com/zurawiki/gptcommit/releases/download/v0.5.7/gptcommit-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 5A27AF5A2A81178D35B5E664BA03CDF5394A7E8E3A6D5BCACACBE5D430D43E3B
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: .\gptcommit.exe
+    PortableCommandAlias: gptcommit
+  InstallerUrl: https://github.com/zurawiki/gptcommit/releases/download/v0.5.7/gptcommit-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 5A27AF5A2A81178D35B5E664BA03CDF5394A7E8E3A6D5BCACACBE5D430D43E3B

```

Comment:
```suggestion
```

Reply:
Validation failed on the x86 package, and x86 packages are validated on 32-bit OSes. So this might be a 64-bit package.

Reply:
Removed the 32-bit build. This is 64-bit only